stdlib/SuiteSparse/test/cholmod.jl

```diff
@@ -855,3 +855,13 @@ end
         @test !issuccess(ldlt!(F, M; check = false))
     end
 end
+
+@testset "Issue #27860" begin
+    for typeA in (Float64, ComplexF64), typeB in (Float64, ComplexF64), transform in (adjoint, transpose)
+        A = sparse(typeA[2.0 0.1; 0.1 2.0])
+        B = randn(typeB, 2, 2)
+        b = randn(typeB, 1, 2)
```

Comment:
This is a `Matrix`, and `transform(b)` is an `Adjoint/Transpose{<:Any,<:AbstractMatrix}`, not `Adjoint/Transpose{<:Any,<:AbstractVector}`.

Reply:
With the `Vector` methods deleted, I’m no longer sure the `b` tests add anything. At least, we don’t seem to test previously untested methods.